conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,16 @@
+-- Products schema
+
+-- !Ups
+
+CREATE TABLE IF NOT EXISTS Product (
+    id          UUID          NOT NULL PRIMARY KEY,
+    name        VARCHAR(100)  NOT NULL,
```

Comment:
There are no requirements for length of a name, remove the constraint

Reply:
removed character length limit